tests/test_smart_ptr.py

```diff
@@ -237,6 +238,44 @@ def test_unique_ptr_arg():
     assert stats.alive() == 0
 
 
+def test_unique_ptr_keep_alive():
+    obj_stats = ConstructorStats.get(m.UniquePtrHeld)
+    c_plain_stats = ConstructorStats.get(m.ContainerPlain)
+    c_keep_stats = ConstructorStats.get(m.ContainerKeepAlive)
+
+    # Try with plain container.
+    obj = m.UniquePtrHeld(1)
```

Comment:
Do we expect any change in behavior when the pointer is NULL? (PR #4 )

Reply:
Done.
(Ran into odd behavior where `reference_internal` was not seeming to work... Will look into that later.)